src/client-configuration.ts

```diff
@@ -120,6 +120,13 @@ export interface ClientConfiguration {
    */
   typecheck?: boolean;
 
+  /**
+   * Enable or disable performance hints. If no value is provided, performance hints will be disabled.
+   * Performance hints will show up in the `summary` field of the QueryInfo object. This ends up as a top
+   * level field in the response object.
```

Comment:
Just some minor nits. Take or leave as you see fit.
```suggestion
   * Enable or disable performance hints. Defaults to disabled.
   * The QueryInfo object includes performance hints in the `summary` field, which is a
   * top-level field in the response object.
```